### PR TITLE
add windows support

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by SimonDanisch** [comment]
btw, there are probably nicer ways of loading and finding the dll/so... If I have time, I can see if I find a nice example!

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Pkg.test("ArrayFire")
 ```
 If you have any issues getting `ArrayFire.jl` to work, please check the Troubleshooting section below. If it still doesn't work, please file an issue. 
 
+#Windows
+Just downdload the [installer](http://arrayfire.com/download/) (you need to create an account).
```
Comment (ranjanan) [inline README.md]: Just fix the typo: download. 

Change line to: Download the official installer from [here](http://arrayfire.com/download/), after creating the account.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ Pkg.test("ArrayFire")
...
+Follow the installation steps and make sure that you include the library directory into the PATH variable as adviced by the installer.
```
Comment (ranjanan) [inline README.md]: ~~adviced~~ -> advised by the installer.

File: README.md (hunk 1 above)
Comment (SimonDanisch) [inline README.md]: oh... must have been really tired :D thanks!

**Pull request merged**